cmd/up/up.go

```diff
@@ -371,12 +373,41 @@ func loadManifestOverrides(dev *model.Dev, upOptions *UpOptions) error {
 		dev.LoadForcePull()
 	}
 
+	if len(upOptions.Envs) > 0 {
+		if err := setEnvVarsFromCmd(dev, upOptions); err != nil {
+			return err
+		}
+	}
+
 	dev.Username = okteto.Context().Username
 	dev.RegistryURL = okteto.Context().Registry
 
 	return nil
 }
 
+func setEnvVarsFromCmd(dev *model.Dev, upOptions *UpOptions) error {
+	envVarsIndexFromManifest := make(map[string]int)
+	for i, manifestEnv := range dev.Environment {
+		envVarsIndexFromManifest[manifestEnv.Name] = i
+	}
+
+	for _, v := range upOptions.Envs {
+		kv := strings.SplitN(v, "=", 2)
+		if len(kv) != 2 {
```

Comment:
Do we support `-e KEY=${NAMESPACE}` expanding env vars?
We also have the notation:
```
environment:
  - KEY
```
Which will create the env var KEY on the container with the value of the local env var
I think its a good use case to add it

Reply:
OK, I will review both proposals